### PR TITLE
Allow for options (filters) on records

### PR DESCRIPTION
**Pull request by fbatista**
This change allows for filters to be applied as params on the records request.

### CODE REVIEW

**Comment by michaelachrisco** [comment]
In order for us to merge the pull request, you must add applicable tests and have them pass travis.

**Comment by fbatista** [comment]
I understand, but i don't have the time to spend on it right now. I will try to do so when i can.

File: lib/knackhq/client.rb [hunk 1]
```diff
@@ -38,13 +38,13 @@ def fields(object)
     end
 
     def records(object, options = {})
-      rows_per_page = options[:rows_per_page] || 25
-      page_number = options[:page_number] || 1
+      rows_per_page = options.delete(:rows_per_page) || 25
```
Comment (josephbridgwaterrowe) [inline lib/knackhq/client.rb:41]: May I suggest something like this:

``` ruby
def records(object, options = {})
  defaults = {
    :rows_per_page => 25,
    :page_number => 1
  }
  hash_request = request
    .objects(object)
    .records
    .get(:params => defaults.merge(options))
  payload = payload_hash(hash_request)
  translate_payload(payload) { payload }
end
```

Maybe a refactor to try once there are tests :smile:

Comment (josephbridgwaterrowe) [inline lib/knackhq/client.rb:41]: Oh, and I forgot to say thanks for the contribution!

Thanks! :cake:

**Pull request closed without merging**